guides/vpn/README.md

```diff
@@ -37,7 +37,20 @@ That is it! You should now be connected to the `Made Tech - VPN`. If you are exp
 
 ### Ubuntu/Debian
 
-https://docs.foxpass.com/docs/foxpass-linux-l2tpipsec
+The Made Tech VPN uses Layer 2 Tunneling Protocol which is not installed by default, but can be installed using:
+
+```bash
+sudo apt install network-manager-l2tp network-manager-l2tp-gnome
+```
+
+Once that's done create the VPN connection by:
```

Comment:
This looks roughly the same as what I did but for reference I didn't need to install anything all required libraries were already there (may have done so earlier though I suppose).

For gateway I used `madetechvpn.uksouth.cloudapp.azure.com`

In PPP Settings I had to check MSCHAP/MSCHAPv2 authentication options to be able to use MPPE

Reply:
thanks @baarkerlounger - I've updated that section